index.css

```diff
@@ -299,6 +308,28 @@ body {
   display: none;
 }
 
+.button:active {
+  color: black;
+}
+
+.error-message {
+  display: flex; 
+  flex-direction: column; 
+  align-items: center; 
+}
+
+.error-title {
+  color: var(--red-dark); 
+  font-size: 16px; 
+  line-height: 24px; 
+  margin: 0px; 
+}
+
+.error-description {
+  font-size: 12px; 
+  line-height: 16px; 
```

Comment:
Mind adding this too? The benefit is for long strings or small screens, it will stay centered if the text wraps to a second line and won't touch the window edges.
```
padding: 0 16px;
text-align: center;
```

I'm also a big fan of max-widths on text like this to force wrap it if it's long but that's not super necessary if we're controlling the length of the message. Just mentioning it for a btw.

Reply:
Added, thank you!